app/models/mixins/supports_feature_mixin.rb

```diff
@@ -113,13 +109,13 @@ def supports_not(feature, reason: nil)
 
     # query the class if the feature is supported or not
     def supports?(feature)
-      public_send("supports_#{feature}?")
+      try("supports_#{feature}?") || false
```

Comment:
Not for this PR, but feels like we should store a lambda/proc or explicit true/false per feature in a hash - and execute the proc in the right context.  That would eliminate the `supports_#{feature}?` method sprawl

Reply:
well, not calling the `supports_feature?` at all and only calling `supports?(:feature)` would allow us to transition to a hash lookup insteam

Reply:
it should be easy enough to find all the callers and fix them to always use `supports?(:feature)`